daemon/logger/jsonfile/jsonfile.go

```diff
@@ -58,3 +59,21 @@ func (lf *JSONLogFile) Close() error {
 	lf.closed = true
 	return nil
 }
+
+// ValidateLogOpt validate log options for json-file log driver
+func ValidateLogOpt(cfg map[string]string) error {
+	for key := range cfg {
+		switch key {
+		case "max-file":
```

Comment:
Could you add all these valid type in a single slice?
And we could check if the input type is in the slice. Then we could make the detailed implemented function decoupled from the specific string, but only a parameter of slice.